Frontend/CO.CDP.OrganisationApp/CacheControlMiddleware.cs

```diff
@@ -0,0 +1,25 @@
+using Microsoft.AspNetCore.Http;
+
+public class CacheControlMiddleware
+{
+    private readonly RequestDelegate _next;
+
+    public CacheControlMiddleware(RequestDelegate next)
+    {
+        _next = next;
+    }
+
+    public async Task Invoke(HttpContext context)
+    {
+        context.Response.OnStarting(() =>
+        {
+            if (!context.Response.Headers.ContainsKey("Cache-Control"))
```

Comment:
Please can you add an extra bit of logic here so that it does not add this header to the static assets - CSS, JS etc. (Anything that comes from wwwroot). I ran this locally and it is adding the header to our site.css.

Reply:
Apart from that, looks good. Thanks 👍🏻